### PR TITLE
Fix broken links in the documentation

### PR DESCRIPTION
**Pull request by amalchuk**

### CODE REVIEW

File: doc/020_installation.md [hunk 1]
```diff
@@ -334,26 +334,26 @@ is copied to the corresponding directory in the Quake II installation.
 
 ### Compiling on Windows with Visual Studio (2015 and newer)
 
-To compile with Visual Studio, CMake is required.
```
Comment (Yamagi) [inline doc/020_installation.md:337]: The double whitespace at the end of a line forces a line break and should not be removed. This is one of the more questionable syntax elements of Markdown...

Comment (amalchuk) [reply doc/020_installation.md:337]: Most editors are trim whitespace in the documents. If a line break is so necessary, it's possible to replace it with:
* `<br>`
* `\`
* double new line?

Comment (Yamagi) [reply doc/020_installation.md:337]: Than the editor config is broken. Double whitespaces are ugly but they are the only way to get a line break in standard Markdown:

* `<br>` requires inline HTML to be enabled and is hard to read in the play text file. We ship the documentation as plain text for the Windows build and some Linux distros do the same.
* `\` isn't standard Markdown and compatibility isn't guaranteed, again it's hard to read in the plain text form.
* A double new line starts a new paragraph.

Comment (amalchuk) [reply doc/020_installation.md]: Okay, got it :octocat:

Comment (amalchuk) [reply doc/020_installation.md]: @Yamagi Let's go to merge?

Comment (Yamagi) [reply doc/020_installation.md]: Yes :)

**Comment by Yamagi** [comment]
Thank you for your work and for incooperating my change requests!

**Pull request merged**